.env.example

```diff
@@ -26,4 +26,5 @@ NORTHSTAR_URI=http://northstar.dev:8000
 NORTHSTAR_API_KEY=abc4324
 
 PHOENIX_URI=https://dev.dosomething.org:8888
+PHOENIX_API_URI=https://staging.beta.dosomething.org/api
```

Comment:
Yeah, plus I think you might need to do some funky local setup stuff to have the Phoenix dev running alongside the Gladiator dev and communicating with each other. Would rather keep staging uri. But in recent merged PR @angaither remove the `api` portion of the uri and is added when constructing the path, so this will like lead to a bug if kept here.